src/mentors.json

```diff
@@ -298,5 +298,34 @@
         "id": "_ArtOfCode"
       }
     ]
+  },
+  {
+    "id": "kamponistullar@gmail.com",
+    "name": "Ogundiji Bolade",
+    "avatar": "https://avatars1.githubusercontent.com/u/19638388?s=460&v=4",
+    "title": "Full Stack Web Developer",
+    "description": "Hi, I'm AdroitCode! Let me know if you need any help.",
+    "country": "Nigeria",
+    "tags": [
+      "php", "frontend", "javascript", "html", "css"
+    ],
+    "channels": [
+      {
+        "type": "email",
+        "id": "kamponistullar@gmail.com"
+      },
+      {
+        "type": "slack",
+        "id": "AdroitCode"
+      },
+      {
+        "type": "twitter",
+        "id": "bollybkampo"
+      },
+      {
+        "type": "linkedin",
```

Comment:
Please follow the [mentor schema](https://github.com/Coding-Coach/find-a-mentor/blob/master/README.md#mentor-schema). Run `yarn test` to make sure everything is ok.

Reply:
Fixed, and done